python/cudf/cudf/core/column/numerical.py

```diff
@@ -316,10 +316,10 @@ def find_and_replace(self, to_replace, replacement, all_nan):
         to_replace_col, replacement_col, replaced = numeric_normalize_types(
             to_replace_col, replacement_col, replaced
         )
-        output = libcudf.replace.replace(
+        output = libcudfxx.replace.replace(
             replaced, to_replace_col, replacement_col
         )
-        return output
+        return column.as_column(output)
```

Comment:
Why do we need a `column.as_column(...)` here now? The call should already be returning a column, no?

Reply:
I was thinking it returns object of type Column, didn't see it was building column depending on the type of the column.